package/yast2-storage-ng.spec

```diff
@@ -47,6 +47,7 @@ BuildRequires:	yast2 >= 4.0.73
 BuildRequires:	yast2-ruby-bindings >= 4.0.6
 BuildRequires:	rubygem(yast-rake)
 BuildRequires:	rubygem(rspec)
+BuildRequires:	rubygem(parallel_tests)
```

Comment:
I see the only one reason to diverge here is the availability of this package. Would it be possible to avoid it? For example, adding some kind of check to ask whether the package exists or not. Because master and SLE-15-GA branches would be exact the same after this PR (and probably during some time), so it would be a pain to maintain the differences in version numbers when the code is the same.

Reply:
That's a good point! :+1: It should be possible to check the target version in the .spec file.

However, currently it seems to be a bit broken: [SLE15-SP1](https://build.suse.de/project/prjconf/SUSE:SLE-15-SP1:GA) defines `%sle_version 150100`, but [Leap 15.1](https://build.opensuse.org/project/prjconf/openSUSE:Leap:15.1) still has
```
%suse_version 1500
%is_opensuse 1
%sle_version 150000
```
which looks wrong to me...

Reply:
Reported in https://bugzilla.suse.com/show_bug.cgi?id=1097123...